pkg/pillar/hypervisor/kvm.go

```diff
@@ -721,6 +778,16 @@ func (ctx KvmContext) Setup(status types.DomainStatus, config types.DomainConfig
 		swtpmCtrlSock = fmt.Sprintf(types.SwtpmCtrlSocketPath, domainName)
 	}
 
+	// Before we start building the domain config, we need to prepare the OVMF settings
+	if config.VirtualizationMode == types.FML {
+		ctx.virtualizationMode = "FML"
```

Comment:
why are we using a string type here?

Reply:
To make it comparable with string constant here https://github.com/lf-edge/eve/blob/72c96a2dca3c6139fde61698f147b2a94c5c0d24/pkg/pillar/hypervisor/kvm.go#L89
Or can we somehow compare with `types.FML` integer const somehow?

Reply:
No nice way to do it =(

Reply:
fyi: https://pkg.go.dev/golang.org/x/tools/cmd/stringer - but perhaps the next time ...